spec/core_ext/process/pause_resume_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'sys-proctable'
+
+RSpec.describe Process do
+  let(:linux) { Gem::Platform.local.os == 'linux' }
+
+  def get_process_status(pid)
+    process = Sys::ProcTable.ps(:pid => pid, :smaps => false, :cgroup => false)
+    linux ? process.state : process.status
+  end
+
+  def paused_status
+    linux ? 'T' : 4
```

Comment:
So, is `4` the status is the `paused` status for Windows or macOS? (or both?)

If it isn't both, does that mean these specs can't run on one of those operating systems?

Reply:
Just Mac. As per my previous comment, these specs will not currently pass on Windows because I can't currently get a process status to verify that it's paused or not. But, since we're not explicitly supporting Windows I was hoping you would let that slide for now.